source/scss/components/_dropdown.scss

```diff
@@ -0,0 +1,12 @@
+.dropdown {
+  padding: 0;
+
+  .dropdown-menu {
+    border: 0;
+    border-radius: 0;
+    box-shadow: 0 4px 5px 2px transparentize($black, .9);
```

Comment:
should have a shadow mixin

Reply:
This shadow was abstracted using variables for now. Issue #45 was created to organize shadows in a better way in the future.